_data/whats-new.yml

```diff
@@ -4,8 +4,52 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Wed Mar 27 18:03:03 2019
+updated: Tue Apr  2 11:41:12 2019
 entries:
+- description: Added a topic about [how to retrieve](https://devdocs.magento.com/guides/v2.3/graphql/get-customer-authorization-token.html)
+    the customer's authorization token for GraphQL queries and mutations.
+  versions: 2.3.1
+  type: New topic
+  date: April  1, 2019
+  link: https://github.com/magento/devdocs/pull/4052
+- description: Added section about mutations in [Resolvers](https://devdocs.magento.com/guides/v2.3/graphql/develop/resolvers.html)
+  versions: 2.3.x
+  type: Major update
+  date: March 30, 2019
+  link: https://github.com/magento/devdocs/pull/4096
+- description: Changed Third-party licenses topics to [Packages](https://devdocs.magento.com/guides/v2.3/release-notes/packages-open-source.html)
+    and added a section with `magento/`-specific dependencies.
+  versions: 2.x
+  type: Major update
+  date: March 29, 2019
+  link: https://github.com/magento/devdocs/pull/4087
+- description: Moved the MFTF docs from `<devdocs root>/mftf/2.3/` to [`<devdocs root>/mftf/docs/`](https://devdocs.magento.com/mftf/docs/introduction.html).<br/>The
+    docs have been moved to the [MFTF repository](https://github.com/magento/magento2-functional-testing-framework).
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: March 28, 2019
+  link: https://github.com/magento/devdocs/pull/3898
+- description: 'Updated the [Cloud Release notes for ece-tools version 2002.0.17](https://devdocs.magento.com/guides/v2.3/cloud/release-notes/cloud-tools.html#v2002017).<br/>-
```

Comment:
This tick at the beginning of "Updated" seems unneeded.

Reply:
It is generated by the native Ruby YAML library, and thus is trusted to be valid.